examples/green-house-migration/legacy/greenhouse/batch/applications.py

```
@@ -0,0 +1,73 @@
+        ("scorecards", "applications/{id}/scorecards", []),
+    ]
+
+    def safe_fetch(
+        self, app_id: int, key: str, path_template: str, default: Any
+    ) -> Tuple[str, Any]:
+        """Safely fetch data from the Greenhouse API, handling errors gracefully."""
+        url_path = path_template.format(id=app_id)
+
+        try:
+            return key, gh_get(url_path)
+        except requests.exceptions.HTTPError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)